mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -6594,6 +6594,26 @@ bool ConstantMaskOp::isAllOnesMask() {
   return true;
 }
 
+static Attribute createBoolSplat(ShapedType ty, bool x) {
+  return SplatElementsAttr::get(ty, BoolAttr::get(ty.getContext(), x));
```

Comment:
Let's not create a free function for this, better inline it or a lambda.

Reply:
random comment: we have some of these utilities in the IR builder... would it make sense to move this one also there or is it too specific?